src/main/webapp/app/shared/type-ahead-search-field/type-ahead-user-search-field.component.html

```diff
@@ -0,0 +1,33 @@
+<label class="mb-1" for="typeahead-user-login">{{ 'artemisApp.dataExport.loginOrName' | artemisTranslate }}</label>
+<div class="search-container">
+    <input
+        id="typeahead-user-login"
+        type="text"
+        class="form-control"
+        [(ngModel)]="loginOrName"
+        [ngbTypeahead]="search"
+        [placeholder]="'artemisApp.dataExport.loginOrName' | artemisTranslate"
+        (input)="onChange()"
```

Comment:
is it really necessary to call onChange? I've never seen this in any other component I think.